src/plugins/intel_gpu/tests/common/opencl_helper_instance.hpp

```diff
@@ -95,6 +93,20 @@ struct OpenCL {
         _queue = cl::CommandQueue(_context, _device, props);
     }
 
+    OpenCL(cl_context context, bool out_of_order_queue = true)
```

Comment:
it's more convenient to construct this object from `cl_context` in [sanity tests for GPU integration](https://github.com/slyalin/openvino/pull/169/files#diff-b271816fd5bbe5038e478a1190ea52801add3ee42febd2f5c1cb3e0bb9450cfaR156), since we can simply request the context from compiled model and construct this class